src/arm/tables.S

```diff
@@ -40,8 +40,7 @@
 
 .global X(mc_subpel_filters)
 .hidden X(mc_subpel_filters)
-const X(mc_subpel_filters), align=3
-X(mc_subpel_filters):
+const X(mc_subpel_filters), export=1, align=3
```

Comment:
No, this is wrong.

To fix the symbol redefinition problem, just remove all the `X(symbolname):` lines. Do that in one single commit first. They are redundant and clang is correctly pointing that out. This part doesn't relate to my patch at all.

Then my patch allows you to get rid of three things:
1) The `.global X(symbolname)`
2) The `.hidden X(symbolname)`
3) The `X()` wrapper in the const macro invocation.

So the correct version of this PR would be:
1) Remove superfluous symbols
2) Add my patch for adding the export parameter
3) Remove all `.global`, `.hidden` and `X()` in const invocations, and add `export=1` in the const invocations.

Using `const X(foo), export=1` is wrong, as `export=1` should imply the same prefix as `X()` adds. If it doesn't, there's something missing still in my patch.

Reply:
Thank you for the review,
I have updated the patchset based on your suggestion, can you please check again.